crypto/bls/Cargo.toml

```diff
@@ -15,7 +15,7 @@ ethereum_hashing = { workspace = true }
 ethereum-types = { workspace = true }
 arbitrary = { workspace = true }
 zeroize = { workspace = true }
-blst = { version = "0.3.3", optional = true }
+blst = { version = "0.3.13", optional = true }
```

Comment:
Updates blst to use 0.3.13 which includes an impl of the Multipoint trait on PublicKey